torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -950,6 +968,13 @@ def call_function(
             should_flatten_outputs=True,
         )
 
+        subgraph_example_value = [
+            proxy.node.meta["example_value"] for proxy in body_r.as_proxy()
+        ]
+        map_example_out = [
+            t.expand(sample_shape[0], *t.shape) for t in subgraph_example_value
```

Comment:
This seems sus, expand will generate a view aliasing with the subgraph example value, but you don't actually want to return a view right?

Reply:
em..I just do a `torch.empty_strided`, does it sound like a good idea? BTW, what might go wrong if we return a view of it?